src/pages/resource/ContributorsPanel.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useReducer } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import mockContributors from '../../utils/testfiles/contributors.json';
+
+import Box from '../../components/Box';
+import TabPanel from '../../components/TabPanel/TabPanel';
+import { contributorReducer } from '../../redux/reducers/contributorReducer';
+import { RootStore } from '../../redux/reducers/rootReducer';
+import Contributor from './contributors/Contributor';
+import ContributorLabel from './contributors/ContributorLabel';
+import ContributorSelector from './contributors/ContributorSelector';
+
+const StyledBox = styled.div`
+  background-color: ${({ theme }) => theme.palette.box.main};
+  display: grid;
+  grid-template-areas: 'icon name institution switch orcid arrows delete';
+  grid-template-columns: 5% 30% 18% 10% 5% 5% 5%;
+`;
+
+interface ContributorsPanelProps {
+  onClick: (event: React.MouseEvent<any>) => void;
+  tabNumber: number;
+}
+
+const ContributorsPanel: React.FC<ContributorsPanelProps> = ({ onClick, tabNumber }) => {
+  const { t } = useTranslation();
+  const errors = useSelector((store: RootStore) => store.errors);
+  const initialState = mockContributors;
+  const [contributors, dispatch] = useReducer(contributorReducer, initialState);
```

Comment:
Noen grunn til å ikke bare bruke `useSelector` for å hente alle contributors her, og bruke `useDispatch` for actions i `Contributor.tsx`? Virker som at `dispatch` hentes ut bare for å bli sendt nedover som prop her

Reply:
vi kan ikke bruke useSelector når vi har en lokal reducer. useSelector henter data fra store, men contributors er ikke lagret i store

Reply:
aha
Syns det blir litt misvisende at `ContributorReducer` ligger i redux folder. Kunne eventuelt flyttet `actions` og `reducers` ut av `redux` folder